code/modules/mob/new_player/sprite_accessories.dm

```diff
@@ -1025,6 +1025,11 @@
 		icon_state = "teshari_mushroom"
 		species_allowed = list(SPECIES_TESHARI)
 
+	teshari_fluffymohawk
+		name = "Teshari Fluffy Mohawk"
+		icon_state = "teshari_fluffymohawk"

```

Comment:
Also these need //VOREStation edit comments

Reply:
I'd personally suggest moving them to sprite_accessories.dm altogether.